{"name": "test_countNoMeasure", "status": "failed", "statusDetails": {"message": "AssertionError", "trace": "self = <Tests.MeasureTest.MeasureCount_test.NewTest testMethod=test_countNoMeasure>\n\n    @allure.testcase(\"Validate measure count with invalid measure\")\n    @allure.description(\"\")\n    @pytest.mark.flaky(reruns=2, reruns_delay=5)\n    @allure.severity(allure.severity_level.CRITICAL)\n    def test_countNoMeasure(self):\n        self.lp.login(user_data['username'], user_data['password'])\n        result = self.lp.verifyLoginSuccessful()\n>       self.ts.markFinal(\"test_validLogin\", result, \"Login was successful\")\n\nMeasureTest/MeasureCount_test.py:71: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <Utils.teststatus.TestStatus object at 0x11765c860>\ntestName = 'test_validLogin', result = False\nresultMessage = 'Login was successful'\n\n    def markFinal(self, testName, result, resultMessage):\n        \"\"\"\n        Mark the final result of the verification point in a test case\n        This needs to be called at least once in a test case\n        This should be final test status of the test case\n        \"\"\"\n        self.setResult(result, resultMessage)\n    \n        if \"FAIL\" in self.resultList:\n            self.log.error(testName +  \" ### TEST FAILED\")\n            self.resultList.clear()\n>           assert True == False\nE           AssertionError\n\n../Utils/teststatus.py:63: AssertionError"}, "attachments": [{"name": "log", "source": "2f63e4dd-87db-41cb-aaf6-46fcb9c5cd2c-attachment.txt", "type": "text/plain"}], "start": 1597146530934, "stop": 1597146558315, "uuid": "4691516d-e41d-4171-9856-0ba584966041", "historyId": "8d40057cce3b6cfc729df83eef46af24", "testCaseId": "e232a189a321b0e48706dc4e4b13797a", "fullName": "MeasureTest.MeasureCount_test.NewTest#test_countNoMeasure", "labels": [{"name": "severity", "value": "critical"}, {"name": "tag", "value": "flaky(reruns=2, reruns_delay=5)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('oneTimeSetUp', 'setUp')"}, {"name": "parentSuite", "value": "MeasureTest"}, {"name": "suite", "value": "MeasureCount_test"}, {"name": "subSuite", "value": "NewTest"}, {"name": "host", "value": "THO40157601L.tenethealth.net"}, {"name": "thread", "value": "16638-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "MeasureTest.MeasureCount_test"}], "links": [{"type": "test_case", "url": "Validate measure count with invalid measure", "name": "Validate measure count with invalid measure"}]}